libs/pages/elewa/home/src/lib/features-elewa-home.module.ts

```diff
@@ -1,9 +1,10 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { HomePageComponent } from './pages/home-page/home-page.component';
+import { CardsModule } from '@elewa-group/features/components/cards';
 
```

Comment:
Order of imports - Local imports come last i.e

import { HomePageComponent } from './pages/home-page/home-page.component'; should be last import